composer.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "reload/bugyield",
+  "description": "A script to sync between Harvest and Jira.",
+  "license": "MIT",
+  "autoload" : {
+    "psr-4": {
+      "": "src/"
+    }
+  },
+  "require": {
+    "php": ">=5.4.0",
+    "gridonic/hapi": "dev-master",
+    "symfony/console": "2.7.2",
+    "symfony/yaml": "2.7.2",
+    "alexkovalevych/jira-api": "1.0.5"
```

Comment:
* I'll change the symfony packages to ~2.7
* The HAPI package has some namespace errors in the 0.1 release (only release so far), so we have to use dev-master
* JIRA-API: I don't know the maintainer, so it's more precaution than anything else :)